pkg/sns/Tests/SnsConnectionFactoryConfigTest.php

```diff
@@ -148,6 +154,24 @@ public static function provideConfigs()
                 'version' => '2010-03-31',
                 'lazy' => false,
                 'endpoint' => 'http://localstack:1111',
+                'topic_arns' => [],
+            ],
+        ];
+
+        yield [
+            ['dsn' => 'sns:?topic_arns=topic1|arn:aws:sns:us-east-1:123456789012:topic1;topic2|arn:aws:sns:us-west-2:123456789012:topic2'],
```

Comment:
what about passing arns as

```
topic_arns[topic1]=arn:aws:sns:us-east-1:123456789012&topic_arns[topic2]=arn:aws:sns:us-west-2:123456789012:topic2
```

that way the standard query parsing function does the job

Reply:
good point, replaced